docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+
+services:
+
+  mariadb:
+    build: docker/database
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: 'mdp-lib'
```

Comment:
See https://github.com/hathitrust/holdings-backend/blob/master/docker-compose.yml#L21 for my latest thinking on how to manage this -- don't need a separate Dockerfile, and we can set a random mysql root password (shouldn't really matter)

Reply:
Works. I'm gonna move docker/database/sql/ to just sql/.